src/main/java/com/compstore/config/CrossOriginConfig.java

```diff
@@ -1,36 +1,41 @@
 package com.compstore.config;
 
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 import org.springframework.web.servlet.config.annotation.CorsRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 @Configuration
-public class CrossOriginConfig implements WebMvcConfigurer {
+public class CrossOriginConfig {
 
     @Value("${cors.allowed-origins}")
-    private static String allowedFrontend;
+    private String allowedFrontend;
```

Comment:
Tak jak pisałeś problemem było używnie pól i klas statycznych. 

Zmieniłem tu też sposób tworzenia beanów - to było niekonieczne, ale akurat takie rozwiązanie znalazłem podczas potwierdzania rozwiązania powyższego